### ISSUE TITLE
Cannot open a large TIFF

### ISSUE THREAD

**Issue by sirius-c** [nomacs/nomacs#303]
**Describe the bug**
Cannot open a particular TIFF file of 1.7 GByte which does open in GIMP 2.8.22

**To Reproduce**
1. Download the file [http://sci.esa.int/science-e-media/img/6d/heic1901a.tif]()
2. Try to open with nomacs
3. Message in the GUI window "Sorry, I could not load...". Terminal output:
```
$ nomacs heic1901a.tif 
[INFO] Hi there
[WARNING] QObject::connect: Cannot connect (null)::runPlugin(DkViewPortInterface*, bool) to nmc::DkControlWidget::setPluginWidget(DkViewPortInterface*, bool)
[WARNING] QObject::connect: Cannot connect (null)::applyPluginChanges(bool) to nmc::DkControlWidget::applyPluginChanges(bool)
[WARNING] QObject::connect: Cannot connect (null)::runPlugin(DkPluginContainer*, const QString&) to nmc::DkViewPort::applyPlugin(DkPluginContainer*, const QString&)
[INFO] local client created in:  3 ms
[INFO] CSS loaded from:  ":/nomacs/stylesheet.css"
[INFO] LAN client created in:  0 ms
[INFO] Initialization takes:  61 ms
TIFFFetchNormalTag: Warning, Incompatible type for "RichTIFFIPTC"; tag ignored.
TIFFFetchNormalTag: Warning, Incompatible type for "RichTIFFIPTC"; tag ignored.
TIFFFetchNormalTag: Warning, Incompatible type for "RichTIFFIPTC"; tag ignored.
TIFFFetchNormalTag: Warning, Incompatible type for "RichTIFFIPTC"; tag ignored.
[WARNING] could not load "/home/ogrimm/heic1901a.tif"
```

**Expected behavior**
Image is displayed.

**Desktop (please complete the following information):**
 - Xubuntu 18.04.1 LTS, 64 bit
- 16 GByte RAM (with GIMP displaying the picture, about 6 GByte are used)
 - nomacs version 3.8
```
$ uname -a
Linux LaptopHP 4.15.0-43-generic #46-Ubuntu SMP Thu Dec 6 14:45:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by diemmarkus** [comment]
Hi,

that's interesting for I can load it on my system (nomacs consumes 2.5GB RAM):
![img-2019-01-24 10 18 04](https://user-images.githubusercontent.com/4182443/51667960-67ac3900-1fc1-11e9-8b1c-cdd2789d1225.png)
But the image width exceeds 32767 which causes an error in the Qt rendering engine (see screenshot). Because of the error message, I am guessing that your issue results from the TIF plugin that is used on your OS. So the bad news is that we are not really able to fix this issue...

best,
markus

**Comment by sirius-c** [comment]
Thanks for following up. Could you advise whom to contact about the possible TIF plugin problem?

**Comment by diemmarkus** [comment]
sorry for the late response... I had a look at this issue again & tested it using nomacs 3.8 (as you stated) - same behavior as you describe. So the issue is definitely caused by nomacs and fixed in the current (3.12) release.

Now there are two options: 
- wait until nomacs 3.12 is released for your OS 
- build it using the ubuntu compile instructions: https://github.com/nomacs/nomacs

**Comment by sirius-c** [comment]
Thanks for fixing!
